lib/ansiblelint/cli.py

```diff
@@ -0,0 +1,180 @@
+                      help="parseable output in the format of pep8")
+    parser.add_option('--parseable-severity', dest='parseable_severity',
+                      default=False,
+                      action='store_true',
+                      help="parseable output including severity of rule")
+    parser.add_option('-r', action='callback', dest='rulesdir',
+                      default=[], type='str', callback=abspath_arg,
+                      help="specify one or more rules directories using "
+                           "one or more -r arguments. Any -r flags override "
+                           "the default rules in %s, unless -R is also used."
+                           % ansiblelint.default_rulesdir)
+    parser.add_option('-R', action='store_true',
+                      default=False,
+                      dest='use_default_rules',
+                      help="Use default rules in %s in addition to any extra "
+                           "rules directories specified with -r. There is "
+                           "no need to specify this if no -r flags are used"
+                           % ansiblelint.default_rulesdir)
+    parser.add_option('-t', dest='tags',
+                      action='append',
+                      default=[],
+                      help="only check rules whose id/tags match these values")
+    parser.add_option('-T', dest='listtags', action='store_true',
+                      help="list all the tags")
+    parser.add_option('-v', dest='verbosity', action='count',
+                      help="Increase verbosity level",
+                      default=0)
+    parser.add_option('-x', dest='skip_list', default=[], action='append',
+                      help="only check rules whose id/tags do not " +
+                      "match these values")
+    parser.add_option('--nocolor', dest='colored',
+                      default=hasattr(sys.stdout, 'isatty') and sys.stdout.isatty(),
+                      action='store_false',
+                      help="disable colored output")
+    parser.add_option('--force-color', dest='colored',
+                      action='store_true',
+                      help="Try force colored output (relying on ansible's code)")
+    parser.add_option('--exclude', dest='exclude_paths', action='callback',
+                      callback=abspath_arg, type=str, default=[],
+                      help='path to directories or files to skip. This option'
+                           ' is repeatable.',
+                      )
+    parser.add_option('-c', dest='config_file',
+                      help='Specify configuration file to use.  Defaults to ".ansible-lint"')
+
+    return parser
+
+
+def merge_config(file_config, cli_config):
+    if file_config:
+        if 'quiet' in file_config:
+            cli_config.quiet = cli_config.quiet or file_config['quiet']
+
+        if 'parseable' in file_config:
+            cli_config.parseable = cli_config.parseable or \
```

Comment:
It is preferable to avoid EOL escaping in favor of using braces. It can be harmful, long-term.